src/data/cluster_situations_de.json

```diff
@@ -0,0 +1,89 @@
+{
+    "section-main": {
+      "headline": {
+        "title": "Clustersituationen",
+        "textblock": [
+          "Du kannst für folgende Situationen ein erhöhtes Ansteckungsrisiko annehmen.",
+          "Vermeide diese soweit möglich.",
+          "Wenn sie unvermeidbar sind, nutze die Corona-Warn-App, um im Falle einer nachträglich erkannten SARS-CoV-2-Infektionskette schnell gewarnt zu werden, oder selbst warnen zu können."
+        ]
+      }
+    },
+    "section-general-listing": {
+      "headline": {
+        "title": "Generell gilt",
+        "sub-headline": " (\"3 Gs (plus)\"):"
```

Comment:
Merging `section-general-listing` and `section-plus-listing` and keeping this translation would make it `5 Gs` 😁

Reply:
Done, see [below](https://github.com/corona-warn-app/cwa-website/pull/347#issuecomment-687819271)